### PR TITLE
Document `expectAsync().toBeAccessible()`

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
Related: https://github.com/blackbaud/skyux-sdk-testing/pull/37

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-expect-async-toBeAccessible](https://developer.blackbaud.com/skyux?_sp.spa=pr-expect-async-toBeAccessible)

File: src/app/learn/get-started/advanced/accessibility-unit-tests/index.html [hunk 1]
```diff
@@ -105,13 +76,11 @@ <h3 class="sky-subsection-heading">
     In the second <sky-code>toBeAccessible</sky-code> parameter, you can adjust configuration settings to turn off accessibility checks. <a href="https://github.com/blackbaud/skyux-sdk-testing/blob/master/src/app/public/a11y/a11y-analyzer.ts">The <sky-code>SkyA11yAnalyzerConfig</sky-code> interface</a> specifies configuration settings, and it turns all checks on by default. To turn off accessibility checks, you set them to <sky-code>false</sky-code>:
```
Comment (blackbaud-johnly) [inline src/app/learn/get-started/advanced/accessibility-unit-tests/index.html:76]:
```suggestion
    You adjust configuration settings to turn off accessibility checks. <a href="https://github.com/blackbaud/skyux-sdk-testing/blob/master/src/app/public/a11y/a11y-analyzer.ts">The <sky-code>SkyA11yAnalyzerConfig</sky-code> interface</a> specifies configuration settings, and it turns all checks on by default. To turn off accessibility checks, you set them to <sky-code>false</sky-code>:
```

File: src/app/learn/get-started/advanced/accessibility-unit-tests/index.html [hunk 2]
```diff
@@ -32,70 +32,41 @@
...
-  <p>
-    Keep in mind that <sky-code>async</sky-code> is also a native JavaScript declaration, so you need to explicitly import the Angular <sky-code>async</sky-code> method at the top of the unit test spec file.
-  </p>
-
-  <sky-code-block>
-    import { async } from ‘@angular/core/testing’;
-  </sky-code-block>
-
   <stache-page-anchor>
     Specify parameters for <sky-code>toBeAccessible</sky-code>
```
Comment (blackbaud-johnly) [inline src/app/learn/get-started/advanced/accessibility-unit-tests/index.html:65]:
```suggestion
    Overwrite accessibility checks
```

Note: I'd also delete the current subheader that says `Overwrite accessibility checks`. If not, I wouldn't make this change. But it doesn't seem like we need the subheader anymore, and it seems like this header is inaccurate now that we aren't talking about plural `parameters`.

File: src/app/learn/get-started/advanced/accessibility-unit-tests/index.html [hunk 3]
```diff
@@ -32,70 +32,41 @@
...
-        });
-      });
-    }));
-  </sky-code-block>
-  <p>
-    For example, for components that change state, you can use this parameter to change state after accessibility checks and then run the accessibility checks again.
+    Finally, you can specify an optional parameter within the <sky-code>toBeAccessible</sky-code> matcher that lets you overwrite <a href="https://github.com/blackbaud/skyux-sdk-testing/blob/master/src/app/public/a11y/a11y-analyzer.ts">the default accessibility checks that <sky-code>SkyA11yAnalyzerConfig</sky-code> specifies</a>.
   </p>
 
   <h3 class="sky-subsection-heading">
```
Comment (blackbaud-johnly) [inline src/app/learn/get-started/advanced/accessibility-unit-tests/index.html:72]: As mentioned above, I'd delete this header now.

File: src/app/learn/get-started/advanced/accessibility-unit-tests/index.html [hunk 4]
```diff
@@ -32,70 +32,41 @@
...
-      fixture.whenStable().then(() => {
-        expect(element).toBeAccessible(() => {
-          // Do something else after the accessibility checks run...
-        });
-      });
-    }));
-  </sky-code-block>
-  <p>
-    For example, for components that change state, you can use this parameter to change state after accessibility checks and then run the accessibility checks again.
+    Finally, you can specify an optional parameter within the <sky-code>toBeAccessible</sky-code> matcher that lets you overwrite <a href="https://github.com/blackbaud/skyux-sdk-testing/blob/master/src/app/public/a11y/a11y-analyzer.ts">the default accessibility checks that <sky-code>SkyA11yAnalyzerConfig</sky-code> specifies</a>.
```
Comment (blackbaud-johnly) [inline src/app/learn/get-started/advanced/accessibility-unit-tests/index.html:69]:
```suggestion
    You can specify an optional parameter within the <sky-code>toBeAccessible</sky-code> matcher to overwrite <a href="https://github.com/blackbaud/skyux-sdk-testing/blob/master/src/app/public/a11y/a11y-analyzer.ts">the default accessibility checks that <sky-code>SkyA11yAnalyzerConfig</sky-code> specifies</a>.
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-expect-async-toBeAccessible](https://developer.blackbaud.com/skyux?_sp.spa=pr-expect-async-toBeAccessible)

File: src/app/learn/get-started/advanced/accessibility-unit-tests/index.html (hunk 3 above)
Comment (Blackbaud-SteveBrush) [reply src/app/learn/get-started/advanced/accessibility-unit-tests/index.html]: Done!

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-expect-async-toBeAccessible](https://developer.blackbaud.com/skyux?_sp.spa=pr-expect-async-toBeAccessible)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**